raphtory-graphql/src/model/graph/graphs.rs

```diff
@@ -1,48 +1,34 @@
-use crate::model::graph::property::GqlProperties;
+use async_graphql::parser::Error;
 use dynamic_graphql::{ResolvedObject, ResolvedObjectFields};
-use itertools::Itertools;
-use raphtory::{
-    db::api::{properties::dyn_props::DynProperties, view::DynamicGraph},
-    prelude::GraphViewOps,
-    search::{into_indexed::DynamicIndexedGraph, IndexedGraph},
-};
 
 #[derive(ResolvedObject)]
 pub(crate) struct GqlGraphs {
-    graphs: Vec<IndexedGraph<DynamicGraph>>,
+    names: Vec<String>,
```

Comment:
again, would it not be more useful to just return the actual path as the name here?